platform/ios/src/MGLMapView.mm

```diff
@@ -1076,20 +1076,15 @@ - (void)setContentInset:(UIEdgeInsets)contentInset animated:(BOOL)animated
         return;
     }
 
-    // After adjusting the content inset, move the center coordinate from the
-    // old frame of reference to the new one represented by the newly set
-    // content inset.
-    CLLocationCoordinate2D oldCenter = self.centerCoordinate;
-
-    _contentInset = contentInset;
-
     if (self.userTrackingMode == MGLUserTrackingModeNone)
     {
         // Don’t call -setCenterCoordinate:, which resets the user tracking mode.
-        [self _setCenterCoordinate:oldCenter animated:animated];
+        [self _setCenterCoordinate:self.centerCoordinate edgePadding:contentInset zoomLevel:self.zoomLevel direction:self.direction duration:animated ? MGLAnimationDuration : 0 animationTimingFunction:nil completionHandler:NULL];
```

Comment:
Use `nil` instead of `NULL`

Reply:
Throughout this file, we initialize block to NULL (not nil).
In other files, nil is used.
Apple's documentation is also [sometimes using NULL](https://developer.apple.com/documentation/uikit/uiview/1622515-animatewithduration), in similar situations.
For the sake of consistency, even though it is an orthogonal change, changed it on other places within this file.

Reply:
`NULL` is fine here.